Src/java/elm/src/main/java/org/cqframework/cql/elm/tracking/ProfileType.java

```diff
@@ -0,0 +1,25 @@
+package org.cqframework.cql.elm.tracking;
+
+import java.util.Collection;
+
+/**
+ * Created by Bryn on 8/22/2016.
```

Comment:
Based on the exchange with Moesel, it would be nice to have some comments here explaining what a `ProfileType` is and when is should be used.